tests/python/relax/test_transform_legalize_ops_manipulate.py

```diff
@@ -1522,5 +1522,45 @@ def main(
     tvm.ir.assert_structural_equal(mod, Expected)
 
 
+def test_layout_transform():
+    transformation = lambda a, b, c: (a, c, b // 3, b % 3)
+    pad_value = 2
+    # fmt: off
+    @I.ir_module
+    class LayoutTransform:
+        @R.function
+        def main(x: R.Tensor((10, 20, 30), "float32")):
```

Comment:
should we also add a test case with symbolic dimensions?

Reply:
Good idea! Have added a symbolic shape test case.